manual/parts/objects.tex

```diff
@@ -173,7 +173,7 @@ \section{System Calls}
     section \ref{sec:faults}) when attempted with other capability types.
 
     In the MCS configuration, \emph{Receive} takes a reply capability---a
-    capability to a reply object--as a parameter.
+    capability to a reply object---as a parameter.
```

Comment:
should also put a space before and after `---` then?

Reply:
depends on the style, the manual doesn't put spaces there -- you'd have either have to update all of them or none of them. I think the change is correct as is.